Code of Conduct.txt

```diff
@@ -0,0 +1,11 @@
+﻿Das BBWF-Project ist ein jedermann offen stehendes Projekt. Die Entwicklung des Projektes geschieht im öffentlichen Raum. Daher ist eine der Öffentlichkeit angepasse Verhaltensart einzuhalten.
+
+Zusammenfassend gelten an ersten Stelle Recht und Gesetz der BRD, des Weiteren die grundlegenden sozialen Regeln der BRD und der "gesunde Menschenverstand".
```

Comment:
Der Grund für die meisten Auseinandersetzungen ist, dass Leute unterschiedliche Vorstellungen von sozialen Regeln und gesundem Menschenverstand haben. Deswegen beschreibt ein CoC meist explizit, was ok ist und was nicht. Das müsst ihr aber nicht selber ausformulieren - ich dachte eher, dass ihr einen existierenden CoC übernehmt. Müsst ihr aber nicht und der hier ist auch weitestgehend in Ordnung (siehe unten, was geändert werden muss).